### PR TITLE
rpma: change the names of the internal API functions

### PR DESCRIPTION
**Pull request by haichangsi**
I am gonna fix lines (80 -> 100) in another PR

---
This change is [Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)

### CODE REVIEW

**Review by ldorau — CHANGES_REQUESTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: 0 of 35 files reviewed, 9 unresolved discussions (waiting on @haichangsi and @Patryk717)

___
*[`-- commits` line 5 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8Obv390gibbcM8Ci6b):*
Rename ... . (a dot at the end - full English sentence, please)



_Code quote:_
```
rpma_conn_req_accept to rpma_conn_new_accept
```
___
*[`-- commits` line 9 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8Oc4pN3CfbDyJX7t-f):*
.



_Code quote:_
```
rpma_conn_req_connect_active to rpma_conn_new_connect
```
___
*[`-- commits` line 13 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8Oc6mT4WgTqmumJsl2):*
.



_Code quote:_
```
rpma_peer_usage_to_access to rpma_peer_usage2access
```
___
*[`-- commits` line 17 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8Oc8bq-_qaNydGiWAo):*
.



_Code quote:_
```
rpma_flush_apm_do to rpma_flush_apm_request
```
___
*[`-- commits` line 21 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OcAPFDbsI8WEBhONW):*
.



_Code quote:_
```
rpma_private_data_discard to rpma_private_data_delete
```
___
*[`-- commits` line 25 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OcCPc8xhAZXPOfXxF):*
.



_Code quote:_
```
rpma_peer_mr_reg to rpma_peer_setup_mr_reg.
```
___
*[`-- commits` line 29 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OcDtc9sw_iAYqvHDq):*
.



_Code quote:_
```
rpma_conn_req_from_id to rpma_conn_req_new_from_id
```
___
*[`-- commits` line 33 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OcFdW1R5eoFZMPDDY):*
.



_Code quote:_
```
rpma_conn_req_from_cm_event to rpma_conn_req_new_from_cm_event
```
___
*[`-- commits` line 37 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OcH2B83oQUZr-9gdw):*
.



_Code quote:_
```
rpma_peer_create_qp to rpma_peer_setup_qp.
```

**Review by ldorau — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: 0 of 35 files reviewed, 10 unresolved discussions (waiting on @haichangsi and @Patryk717)

___
*[`-- commits` line 17 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OezP_6X8UTK_BsIRG):*
Use `rpma_flush_apm_execute` instead, please.



_Code quote:_
```
rpma_flush_apm_request
```

**Review by haichangsi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: 0 of 35 files reviewed, 10 unresolved discussions (waiting on @ldorau and @Patryk717)

___
*[`-- commits` line 5 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8Obv390gibbcM8Ci6b):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

Rename ... . (a dot at the end - full English sentence, please)
</blockquote></details>

Done.
___
*[`-- commits` line 9 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8Oc4pN3CfbDyJX7t-f):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

.
</blockquote></details>

Done.
___
*[`-- commits` line 13 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8Oc6mT4WgTqmumJsl2):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

.
</blockquote></details>

Done.
___
*[`-- commits` line 17 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8Oc8bq-_qaNydGiWAo):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

.
</blockquote></details>

Done.
___
*[`-- commits` line 17 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OezP_6X8UTK_BsIRG):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

Use `rpma_flush_apm_execute` instead, please.
</blockquote></details>

Done.
___
*[`-- commits` line 21 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OcAPFDbsI8WEBhONW):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

.
</blockquote></details>

Done.
___
*[`-- commits` line 25 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OcCPc8xhAZXPOfXxF):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

.
</blockquote></details>

Done.
___
*[`-- commits` line 29 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OcDtc9sw_iAYqvHDq):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

.
</blockquote></details>

Done.
___
*[`-- commits` line 33 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OcFdW1R5eoFZMPDDY):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

.
</blockquote></details>

Done.
___
*[`-- commits` line 37 at r1](https://reviewable.io/reviews/pmem/rpma/1963#-N8OcH2B83oQUZr-9gdw):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

.
</blockquote></details>

Done.

**Review by ldorau — COMMENTED** [review]
> Reviewed 30 of 35 files at r1, 5 of 5 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @Patryk717)

**Review by ldorau — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963#-:-N8TElC9Ez1z5lzh8ebJ:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @haichangsi and @Patryk717)

**Review by ldorau — CHANGES_REQUESTED** [review]
> Reviewed 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: all files reviewed, 2 unresolved discussions (waiting on @haichangsi and @Patryk717)

*[a discussion](https://reviewable.io/reviews/pmem/rpma/1963#-N8hUMwd84a0s3b-fn1l:-N8hUMwe0E_4NbTl31Nb:b2x6w81) (no related file):*
https://app.circleci.com/pipelines/github/haichangsi/rpma/282/workflows/5b4b2185-0a69-445c-8ffb-4685242be07f/jobs/283

___

**Review by ldorau — COMMENTED** [review]
> Reviewed 10 of 10 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: all files reviewed, 2 unresolved discussions (waiting on @haichangsi and @Patryk717)

**Review by ldorau — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: all files reviewed, 1 unresolved discussion (waiting on @haichangsi and @Patryk717)

*[a discussion](https://reviewable.io/reviews/pmem/rpma/1963#-N8xPVu_0FWw6DD52fjU:-N8xPVu_0FWw6DD52fjV:b-ujmq67) (no related file):*
Rebase please

___

**Review by haichangsi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: 25 of 40 files reviewed, 1 unresolved discussion (waiting on @ldorau and @Patryk717)

*[a discussion](https://reviewable.io/reviews/pmem/rpma/1963#-N8xPVu_0FWw6DD52fjU:-N9710ws7YAI72H_PhEZ:b-896fix) (no related file):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

Rebase please
</blockquote></details>

Done.

___

**Review by ldorau — APPROVED** [review]
> Reviewed 15 of 15 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @Patryk717)

File: src/conn_req.c [hunk 1]
```diff
@@ -128,16 +128,8 @@ rpma_conn_req_new_from_id(struct rpma_peer *peer, struct rdma_cm_id *id,
 			goto err_rpma_cq_delete;
 	}
 
-<<<<<<< HEAD
```
Comment (ldorau) [inline src/conn_req.c]: Merge conflict committed.

File: src/peer.c [hunk 2]
```diff
@@ -85,7 +85,6 @@ rpma_peer_usage2access(struct rpma_peer *peer, int usage)
 }
 
 /*
-<<<<<<< HEAD
```
Comment (ldorau) [inline src/peer.c]: Merge conflict committed.

File: src/peer.h [hunk 3]
```diff
@@ -1,14 +1,6 @@
 /* SPDX-License-Identifier: BSD-3-Clause */
-<<<<<<< HEAD
```
Comment (ldorau) [inline src/peer.h]: Merge conflict committed.

File: tests/unit/common/mocks-rpma-peer.c [hunk 4]
```diff
@@ -1,14 +1,6 @@
 // SPDX-License-Identifier: BSD-3-Clause
-<<<<<<< HEAD
```
Comment (ldorau) [inline tests/unit/common/mocks-rpma-peer.c]: Merge conflict committed.

**Review by ldorau — CHANGES_REQUESTED** [review]

File: src/conn_req.c [hunk 5]
```diff
@@ -128,8 +128,16 @@ rpma_conn_req_new_from_id(struct rpma_peer *peer, struct rdma_cm_id *id,
 			goto err_rpma_cq_delete;
 	}
 
+<<<<<<< HEAD
```
Comment (ldorau) [inline src/conn_req.c]: Merge conflict committed.

File: src/peer.c [hunk 6]
```diff
@@ -85,6 +85,7 @@ rpma_peer_usage2access(struct rpma_peer *peer, int usage)
 }
 
 /*
+<<<<<<< HEAD
```
Comment (ldorau) [inline src/peer.c]: Merge conflict committed.

**Review by ldorau — CHANGES_REQUESTED** [review]

File: src/peer.h [hunk 7]
```diff
@@ -1,6 +1,14 @@
 /* SPDX-License-Identifier: BSD-3-Clause */
+<<<<<<< HEAD
```
Comment (ldorau) [inline src/peer.h]: Merge conflict committed.

File: tests/unit/common/mocks-rpma-peer.c [hunk 8]
```diff
@@ -1,6 +1,14 @@
 // SPDX-License-Identifier: BSD-3-Clause
+<<<<<<< HEAD
```
Comment (ldorau) [inline tests/unit/common/mocks-rpma-peer.c]: Merge conflict committed.

File: tests/unit/conn_req/conn_req-from_cm_event.c [hunk 9]
```diff
@@ -595,16 +595,8 @@ from_cm_event__private_data_store_E_NOMEM_subsequent_ERRNO2(void **cstate_ptr)
 	will_return(rpma_cq_delete, MOCK_ERRNO2); /* second or third error */
 	expect_value(rdma_destroy_id, id, &cstate->id);
 	will_return(rdma_destroy_id, MOCK_ERRNO2); /* third or fourth error */
-<<<<<<< HEAD
```
Comment (ldorau) [inline tests/unit/conn_req/conn_req-from_cm_event.c]: Merge conflict committed.

**Review by ldorau — CHANGES_REQUESTED** [review]

File: tests/unit/conn_req/conn_req-from_cm_event.c [hunk 10]
```diff
@@ -595,8 +595,16 @@ from_cm_event__private_data_store_E_NOMEM_subsequent_ERRNO2(void **cstate_ptr)
 	will_return(rpma_cq_delete, MOCK_ERRNO2); /* second or third error */
 	expect_value(rdma_destroy_id, id, &cstate->id);
 	will_return(rdma_destroy_id, MOCK_ERRNO2); /* third or fourth error */
+<<<<<<< HEAD
```
Comment (ldorau) [inline tests/unit/conn_req/conn_req-from_cm_event.c]: Merge conflict committed.

**Review by ldorau — CHANGES_REQUESTED** [review]

**Review by ldorau — CHANGES_REQUESTED** [review]
:lgtm_cancel: 

> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963#-:-N977LRg9BjypDL-WZS9:b8b79qu)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @Patryk717)

**Review by ldorau — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: all files reviewed, 5 unresolved discussions (waiting on @haichangsi and @Patryk717)

___
*[`-- commits` line 19 at r5](https://reviewable.io/reviews/pmem/rpma/1963#-N979KWg9dZbjsUrojF_):*
Contains unresolved merge conflicts.



_Code quote:_
```
82a7be1: rpma: change the name of the rpma_private_data_discard()
```
___
*[`-- commits` line 23 at r5](https://reviewable.io/reviews/pmem/rpma/1963#-N979_Us4OE6lcShb27d):*
Contains unresolved merge conflicts.



_Code quote:_
```
f2dbd1a: rpma: change the name of the rpma_peer_mr_reg()
```
___
*[`-- commits` line 27 at r5](https://reviewable.io/reviews/pmem/rpma/1963#-N979febCPOPfhVl74fF):*
Contains unresolved merge conflicts.



_Code quote:_
```
04299b4: rpma: change the name of the rpma_conn_req_from_id
```
___
*[`-- commits` line 31 at r5](https://reviewable.io/reviews/pmem/rpma/1963#-N979rFD98noJoNCgiwe):*
Does not build.



_Code quote:_
```
5e370c0: rpma: change the name of the rpma_conn_req_from_cm_event
```
___
*[`-- commits` line 35 at r5](https://reviewable.io/reviews/pmem/rpma/1963#-N979yEqBfuGTaUyyzZm):*
Does not build.



_Code quote:_
```
6905d7f: rpma: change the name of the rpma_peer_create_qp
```

**Review by haichangsi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: all files reviewed (commit messages unreviewed), 5 unresolved discussions (waiting on @ldorau and @Patryk717)

___
*[`-- commits` line 19 at r5](https://reviewable.io/reviews/pmem/rpma/1963#-N979KWg9dZbjsUrojF_):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

Contains unresolved merge conflicts.
</blockquote></details>

Done.
___
*[`-- commits` line 23 at r5](https://reviewable.io/reviews/pmem/rpma/1963#-N979_Us4OE6lcShb27d):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

Contains unresolved merge conflicts.
</blockquote></details>

Done.
___
*[`-- commits` line 27 at r5](https://reviewable.io/reviews/pmem/rpma/1963#-N979febCPOPfhVl74fF):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

Contains unresolved merge conflicts.
</blockquote></details>

Done.
___
*[`-- commits` line 31 at r5](https://reviewable.io/reviews/pmem/rpma/1963#-N979rFD98noJoNCgiwe):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

Does not build.
</blockquote></details>

Done.
___
*[`-- commits` line 35 at r5](https://reviewable.io/reviews/pmem/rpma/1963#-N979yEqBfuGTaUyyzZm):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

Does not build.
</blockquote></details>

Done.

**Review by ldorau — CHANGES_REQUESTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: all files reviewed, 1 unresolved discussion (waiting on @haichangsi and @Patryk717)

___
*[`-- commits` line 10 at r6](https://reviewable.io/reviews/pmem/rpma/1963#-N9BAXe30AyRso6lR85D):*
Please add `()` as above.



_Code quote:_
```
  Rename rpma_conn_req_from_id to rpma_conn_req_new_from_id.
  Rename rpma_conn_req_from_cm_event to rpma_conn_req_new_from_cm_event.
  Rename rpma_peer_create_qp to rpma_peer_setup_qp.
```

**Review by haichangsi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: all files reviewed, 1 unresolved discussion (waiting on @ldorau and @Patryk717)

___
*[`-- commits` line 10 at r6](https://reviewable.io/reviews/pmem/rpma/1963#-N9BAXe30AyRso6lR85D):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

Please add `()` as above.
</blockquote></details>

Done.

**Review by ldorau — APPROVED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @Patryk717)

**Review by ldorau — APPROVED** [review]
> Reviewed 1 of 1 files at r8, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @Patryk717)

**Review by ldorau — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1963#-:-N9BOwUYEaBFePPjkvOQ:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @Patryk717)

**Pull request merged**